scripts/controllers.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+angular.module('wireframeApp')
+  .controller('GuitarGalleryController', ['$scope', 'guitarListFactory', function($scope, guitarListFactory) {
+    $scope.guitars = guitarListFactory.getGuitars();
+    $scope.guitarFilter = {
+      filtByBrand: '',
+      filtByMultiscale: '',
+      filtByStrings: '',
+      filtByScale: '24.5',
+      filtByPrice: '10000'  
+    };
+    $scope.shortDescription = function(description) {
+      var text = description.substr(0, 50) + "...";
+      return text;
+    };
+
+    $scope.uncheck = function (event) {
+      if ($scope.checked === event.target.value) {
```

Comment:
If the type of $scope.checked is boolean you needn't to write `if ($scope.checked === event.target.value)`. Just `if ($scope.checked)`